dev/PushNotifications/PushNotificationsLongRunningTask.ProxyStub/NotificationsLongRunningProcess.idl

```diff
@@ -11,7 +11,7 @@ import "ocidl.idl";
 [pointer_default(unique)]
 interface INotificationsLongRunningPlatform : IUnknown
 {
-    HRESULT RegisterFullTrustApplication([in] LPCWSTR processName, [in] GUID remoteId, [out] GUID* appId);
+    HRESULT RegisterFullTrustApplication([in] LPCWSTR processName, [in] GUID remoteId, [out] LPWSTR* appId);
```

Comment:
PWSTR. Not LPWSTR

Reply:
PWSTR can't used as a parameter. You would have to use [out, string] PWSTR.

Reply:
What's wrong with adding [string]?